manifester/manifester.py

```diff
@@ -33,7 +33,10 @@ def __init__(
         **kwargs,
     ):
         if minimal_init:
-            self.offline_token = settings.get("offline_token")
+            if kwargs.get("offline_token") is not None:
+                self.offline_token = kwargs.get("offline_token")
+            else:
+                self.offline_token = settings.get("offline_token")
```

Comment:
Currently, with validators disabled, we might bump into `self.offline_token` being none if none of kwargs and settings contain the offline token. Should we raise an exception in that case?
```suggestion
            if not self.offline_token := kwargs.get("offline_token", settings.get("offline_token")):
                raise Exception("Offline token was not set")  # to be replaced with less general 
```

Reply:
Ack, good idea. I will make this change.

Reply:
@ogajduse It turns out that `:=` can't be used to set instance attributes. Doing so throws `SyntaxError: cannot use assignment expressions with attribute`. But I've added another condition to catch this. I also added an exception that prints the error text when generating an access token fails, which should provide more visibility into failures at that stage.